src/features/CourtCaseList/CourtCaseList.tsx

```diff
@@ -107,112 +70,18 @@ const CourtCaseList: React.FC<Props> = ({ courtCases, order = "asc", currentUser
-              {defendantName}
-              <br />
-              <ResolvedTag isResolved={resolutionTimestamp !== null} />
-            </Link>
-          </Table.Cell>
-          <Table.Cell>
-            <DateTime date={courtDate} dateFormat={displayedDateFormat} />
-          </Table.Cell>
-          <Table.Cell>{courtName}</Table.Cell>
-          <Table.Cell>{ptiurn}</Table.Cell>
-          <Table.Cell>
-            <UrgentTag isUrgent={isUrgent} />
-          </Table.Cell>
-          <Table.Cell>
-            <NotesTag notes={notes} />
-          </Table.Cell>
-          <Table.Cell>
-            {Object.keys(exceptions).map((code, codeId) => (
-              <GridRow key={`exception_${codeId}`}>
-                {code}
-                <b>&nbsp;{exceptions[code] > 1 ? `(${exceptions[code]})` : ""}</b>
-              </GridRow>
-            ))}
-          </Table.Cell>
-          <Table.Cell>
-            {errorLockedByUsername && canUnlockCase(errorLockedByUsername) ? (
-              <LockedByTag
-                lockedBy={errorLockedByUsername}
-                unlockPath={unlockCaseWithReasonPath("Exception", `${errorId}`)}
-              />
-            ) : (
-              <LockedByTag lockedBy={errorLockedByUsername} />
-            )}
-          </Table.Cell>
-        </Table.Row>
-      )
-
-      if (triggers.length > 0) {
-        tableBody.push(
-          <Table.Row key={`triggers-row-${idx}`} className={classes.triggersRow}>
-            <Table.Cell>
-              <ConditionalRender isRendered={!!triggerLockedByUsername}>
-                <Image src={"/bichard/assets/images/lock.svg"} width={20} height={20} alt="Lock icon" />
-              </ConditionalRender>
-            </Table.Cell>
-            <Table.Cell />
-            <Table.Cell />
-            <Table.Cell />
-            <Table.Cell />
-            <Table.Cell />
-            <Table.Cell />
-            <Table.Cell>
-              {triggers?.map((trigger, triggerId) => (
-                <GridRow key={`trigger_${triggerId}`}>{getTriggerWithDescription(trigger.triggerCode)}</GridRow>
-              ))}
-            </Table.Cell>
-            <Table.Cell>
-              {triggerLockedByUsername && canUnlockCase(triggerLockedByUsername) ? (
-                <LockedByTag
-                  lockedBy={triggerLockedByUsername}
-                  unlockPath={unlockCaseWithReasonPath("Trigger", `${errorId}`)}
-                />
-              ) : (
-                <LockedByTag lockedBy={triggerLockedByUsername} />
-              )}
-            </Table.Cell>
-          </Table.Row>
-        )
-      }
-    }
-  )
 
   if (courtCases.length === 0) {
     return <Paragraph>{"There are no court cases to show"}</Paragraph>
   }
 
-  return <Table head={tableHead}>{tableBody}</Table>
+  return (
+    <Table head={tableHead}>
+      {courtCases.map((courtCase, idx) => (
```

Comment:
I think that using an `index` as a key is not recommended. Please see https://reactjs.org/docs/lists-and-keys.html (the section starting with "_We don’t recommend using indexes for keys (...)"_.

Reply:
Can you recommend an alternative method? 

In that article it states that it is not recommended but can be used as a last resort if there are no alternative stable ID's. I think it would be difficult and will add necessary complexity to generate unique IDs for each case and therefore in this use-case I believe mapping with indexes is fit for purpose.

Reply:
I think, ideally you'd want to use uniq key that are more meaningful for accessibility reason, but you can use the index to generate a uniq key for instance `court-case-{idx}`

Reply:
What is `courtCase`? Is that unique? I would try some kind of unique text + maybe `index` to make sure. If we can't figure out anything better, I'd leave if as it is.